Sources_v3/StreamChat/APIClient/Endpoints/Payloads/GuestUserTokenPayload_Tests.swift

```diff
@@ -7,7 +7,6 @@ import XCTest
 
 final class GuestUserTokenPayload_Tests: XCTestCase {
     let guestUserDefaultExtraDataJSON = XCTestCase.mockData(fromFile: "GuestUser+DefaultExtraData")
-    let guestUserNoExtraDataJSON = XCTestCase.mockData(fromFile: "GuestUser+NoExtraData")
     let guestUserCustomExtraDataJSON = XCTestCase.mockData(fromFile: "GuestUser+CustomExtraData")
     let guestUserInvalidTokenJSON = XCTestCase.mockData(fromFile: "GuestUser+InvalidToken")
```

Comment:
when `DefaultExtraData` became `NoExtraData`, should the `defaultExtraData` or `noExtraData` tests be removed?

![image](https://user-images.githubusercontent.com/6388510/105365321-6ccb0f80-5bfe-11eb-8672-951cdb48730b.png)